### DISCUSSION TITLE
IDE Query

### DISCUSSION THREAD

**Discussion by Darshposhiya29** [Cyfrin/foundry-full-course-cu#166] [Q&A] [answered]
How to change keyboard shortcuts in remix?

**Comment by ChefAharoni** [comment, marked as answer]
As far as I know, you **cannot** change the default keyboard shortcuts in remix.

It is however to use remix only at the start of your Solidity learning journey, and perform the major coding in a more powerful IDE, like VS Code, where you can fully control your shortcuts and many other things.

**Reply by Darshposhiya29** [reply]
Okay,
Thanks for the help

**Reply by cromewar** [reply]
you are kinda able to change the shortcuts but is not a good idea, as @ChefAharoni mentioned you will not be using REMIX for professional development anyways.
check this: https://ethereum.stackexchange.com/questions/99312/remix-ide-shortcuts
